cmake/modules/AddSwift.cmake

```diff
@@ -88,6 +88,9 @@ function(_add_host_variant_c_compile_link_flags name)
       MACCATALYST_BUILD_FLAVOR ""
       DEPLOYMENT_VERSION "${DEPLOYMENT_VERSION}")
     target_compile_options(${name} PRIVATE -target;${target})
+    if(SWIFT_HOST_VARIANT_SDK IN_LIST SWIFT_APPLE_PLATFORMS)
```

Comment:
The condition isn't needed, the check on L86 is perfect to control this.

Reply:
Thanks, this was not clear to me -- tackled in 90b164fabf0